### ISSUE TITLE
Grammar error(s)

### ISSUE THREAD

**Issue by sherimickelson** [NCAR/python-tutorial#33]
On the landing page:
"or you are tired of tutorials **the** leap into advanced third-party packages"
Should "the" be "that"?  Or something else?

**Comment by mnlevy1981** [comment]
@sherimickelson -- not sure if you got a notification, but https://github.com/NCAR/ncar-python-tutorial is the old version of the tutorial... so I moved this to the updated repository.

**Comment by sherimickelson** [comment]
Under the "Why Python?" section under "Objected Oriented languages":
1. Should it be Object not Objected?
2. "For every custom datatypes". Should "datatypes" be plural?

**Comment by sherimickelson** [comment]
Under the "Creating a Data Dictionary" section.
Should
> This is intended to pick off right where “Reading in a .txt File” left off - you had just commited your new script file that reads in the data from file as a string.

be
This is intended to pick off right where “Reading in a .txt File” left off - you had just commited your new script file that reads in the data from **a** file as a string.

**Comment by sherimickelson** [comment]
Under the "Creating a Data Dictionary" section:

> Then with the datafile open for reading capabilities, we are going to write two separate for loops. A for loop is used for iterating over a sequence (such as a list). It is important to note the syntax of Python for loops: the : at the end of for for line, ...

Not sure what is meant by "at the end of for for line"

**Comment by sherimickelson** [comment]
> Now, to practice nested indexing,get the 5th, first 5, and every other column of rows 8 in data.

Should it be
Now, to practice nested indexing,get the 5th, first 5, and every other column of row 8 in data.
(change rows to row)?

**Comment by sherimickelson** [comment]
In step number 11:

> Initializing the data dict now includes a for loop, where for each variable specified in columns that key is initialized pointing to an empty list. This is the first time you have looped over a dict and added key-value pairs to a dict via assignment.

I'm not following this part of the sentence
"where for each variable specified in columns that key is initialized pointing to an empty list." 
Maybe reword it or maybe it needs a comma between "columns" and "that"?

**Comment by sherimickelson** [comment]
Misspelling in step number 11:

> When reading and parsing the file, you created your first nested for loop. For every line of the datafile, split that line - and then for every desired variable in the columns dict (date, time, tempout): grab the datum from the current split line with the specified index (0, 1, 2), use the dict.get() method to find the desired datatype if **specired** (avoiding key-not-found errors and defaulting to str if unspecified), convert the datum to the desired datatype, and append the datum to the list associated with each column key within the data dict.

specired

**Comment by sherimickelson** [comment]
In the "Writing Functions" section in step number 1:

> You’ve read the temperature data into the tempout variable, but to do this calculation, you also needto read the windspeed variable from column 7.

There needs to be a space in between "need" and "to" (needto).

**Comment by sherimickelson** [comment]
Step number 3:

> To indicate a function in python you type def for define, the name of your function, and then in parenthesis the input arguments of that function, followed by a colon. On the next lines tab-indented is the code of your function, and your return value.

Maybe reword:
On the next lines tab-indented is the code of your function, and your return value.
?

**Comment by sherimickelson** [comment]
Step number 3:

> Here is your first introduction math operators in Python.

Maybe "Here is your first introduction **to** math operators in Python."?

**Comment by sherimickelson** [comment]
Step number 3:

> And then let’s compute a new list with windchill data atthe bottom of mysci.py:

There needs to be a space in between "at" and "the".

**Comment by sherimickelson** [comment]
In step number 5:

> Then, in a DEBUG section at the end of your script to compare the two different values (from data and computed by our function):

It seems like this sentence is missing something?

**Comment by sherimickelson** [comment]
In step number 5:

> Using f-\strings with float formatting you can determine the precision with which to print the values to.

is the "f-\strings" what you intended?

**Comment by jukent** [comment]
Thanks at @sherimickelson A lot of these were hard to see!
